apps/agreement/contracts/Agreement.sol

```diff
@@ -114,249 +87,157 @@ contract Agreement is IArbitrable, IForwarder, AragonApp {
     }
 
     struct Action {
```

Comment:
Since this is now done on every disputable action, I wonder if we can shave some gas off of this and push it to be more "back heavy" (either on challenge or dispute creation).